src/code/z_player_lib.c

```diff
@@ -1798,49 +1987,399 @@ void Player_DrawZoraShield(PlayState* play, Player* player) {
     CLOSE_DISPS(play->state.gfxCtx);
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_player_lib/func_80125500.s")
+void func_80125500(PlayState* play, Player* player, s32 limbIndex, Vec3f* pos, Vec3s* rot) {
```

Comment:
This looks like the foot placement function in z_actor except it uses auxiliary stuff, could we name it based on that?

Reply:
`Player_AdjustLegs`, `Player_PositionLegs`? Or go down the whole inverse kinematics route or whatever it's called?

Reply:
Yes. Do you want to go with `Player_AdjustLegs` and `PlayerAdjustLeg`?
I would prefer to avoid having a pair of functions which the only difference is a single letter

Reply:
You could do `BothLegs` and/or `SingleLeg`?

Reply:
I don't like `BothLegs` for this function since it doesn't do both at the same time, it is mainly a wrapper for calling the other function.
I'll name the function that actually implements the logic as `Player_AdjustSingleLeg` in the meanwhile